frontend/ui/isolanguage.lua

```diff
@@ -0,0 +1,530 @@
+local _ = require("gettext")
+
+local locale_lang_map = {
+    -- from https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
+    ab = "Abkhazian",
+    aa = "Afar",
+    af = "Afrikaans",
+    ak = "Akan",
+    sq = "Albanian",
+    am = "Amharic",
+    ar = "Arabic",
+    an = "Aragonese",
+    hy = "Armenian",
```

Comment:
I won't ask you to change it unless you want to or agree with me, but I find the English order a bit confusing here.